src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -95,17 +98,43 @@ export class ResultsViewPageStore {
         ],
         invoke: () => {
             const clinicalDataSingleStudyFilter = {attributeIds: ["CANCER_TYPE", "CANCER_TYPE_DETAILED"], ids: this.sampleIds.result};
-            return fetchClinicalDataInStudy(this.studyId, clinicalDataSingleStudyFilter)
+            return fetchClinicalDataInStudy(this.studyId, clinicalDataSingleStudyFilter, 'SAMPLE')
         }
     }, []);
 
+    readonly clinicalDataForPatients = remoteData({
+        await: () => [
+            this.patientIds
+        ],
+        invoke: () => {
+            const clinicalDataSingleStudyFilter = {attributeIds: ["12_245_PARTC_CONSENTED"], ids: this.patientIds.result};
+            return fetchClinicalDataInStudy(this.studyId, clinicalDataSingleStudyFilter, 'PATIENT')
+        }
+    }, []);
+
+    readonly mskImpactGermlineConsentedPatientIds = remoteData({
```

Comment:
@alisman I imagine you would want to add clinical attributes to this function the more we end up needing? To prevent doing multiple calls in the future. The `clinicalDataForSamples` is similar in that it only gets `CANCER_TYPE` and `CANCER_TYPE_DETAILED`. Not sure how to best indicate